src/weatherpy/presenter/utils.py

```diff
@@ -0,0 +1,39 @@
+from typing import TypeAlias
+
+from weatherpy.presenter.symbols import DEGREE, Wind
+
+Number: TypeAlias = int | float
+
+
+def get_wind_direction(deg: int) -> Wind:
+    if 0 <= deg < 22.5:
```

Comment:
(nonblocking; nitpicking)
```suggestion
    variations = [(lambda deg: 0 <= deg < 22.5, Wind.N), [lambda deg: 22.5 <= deg < 67.5, Wind.NE], ...]
    for variation in variations:
        if variation[0](deg) is True:
            return variation[1]
```